libraries/c_sdk/standard/mqtt/src/private/iot_mqtt_internal.h

```diff
@@ -450,10 +453,10 @@ typedef struct connContextMapping
     StaticSemaphore_t contextMutexStorage;                                 /**< @brief Static storage for Mutex for synchronization of MQTT Context. */
     SemaphoreHandle_t contextMutex;                                        /**< @brief Mutex for synchronization of network buffer as the same buffer can be used my multiple applications. */
     uint8_t buffer[ NETWORK_BUFFER_SIZE ];                                 /**< @brief Network Buffer used to send packets on the network. This will be used by MQTT context defined above. */
-    NetworkContext_t networkContext;                                       /**< @brief Network Context used to send packets on the network. This will be used by MQTT context defined above. */
```

Comment:
Should this really be removed? Shouldn't each mapping have both a context and params?

Reply:
Having both networkContext and the MQTTTransportParams seems a bit confusing, following other c files pattern I would recommend make networkContext declared in iot_mqtt_api.c